packages/pharos/src/components/image-card/pharos-image-card.ts

```diff
@@ -463,7 +459,6 @@ export class PharosImageCard extends ScopedRegistryMixin(FocusMixin(PharosElemen
   private _isCheckboxDisplayed() {
     return (
       this._isSubtleSelectHover() ||
-      this._isSubtleAndSelectable() ||
```

Comment:
I delete this because I think `_isSubtleAndSelectable()` already covers this case. The checkbox is displayed when it is in subtle select mode and user hovers on it. 
Adding `_isSubtleAndSelectable()` creates a bug because when it is in subtle select mode, `_isSubtleAndSelectable()` is `true` even no one hovers on the layout and the checkbox doesn't show up. That makes `_isCheckboxDisplayed()` to be `true` while no checkbox actually shows up on the page.

Reply:
This is very helpful, thank you!